Sources/MapboxCoreNavigation/NavigationSettings.swift

```diff
@@ -117,21 +130,24 @@ public class NavigationSettings {
      `TileStoreConfiguration` for more details.
        - routingProviderSource: Configures the type of routing to be used by various SDK objects when providing route calculations. Use this value to configure usage of onlive vs. offline data for routing.
        - alternativeRouteDetectionStrategy: Configures how `AlternativeRoute`s will be detected during navigation process.
+       - utilizeSensorData: Enables using sensors data to improve positioning.
      */
     public func initialize(directions: Directions,
                            tileStoreConfiguration: TileStoreConfiguration,
                            routingProviderSource: RoutingProviderSource = .hybrid,
-                           alternativeRouteDetectionStrategy: AlternativeRouteDetectionStrategy? = .init()) {
+                           alternativeRouteDetectionStrategy: AlternativeRouteDetectionStrategy? = .init(),
+                           utilizeSensorData: Bool = false) {
```

Comment:
What if instead of adding new parameters to this function we add a new method that accepts a structure with all parameters and defaults and deprecate this one?

Reply:
this would help avoid bumping the base api commit, but it won't make configuration easier for user. We are tracking SDK settings overhaul, but it seems to be an invasive change we would want to do in next major version